test/index.test.js

```diff
@@ -0,0 +1,89 @@
+const puppeteer = require("puppeteer");
+
+let browser;
+
+beforeAll(async () => {
+  browser = await puppeteer.launch({});
+});
+
+let page;
+
```

Comment:
there should be a way to set these as global variables in your `package.json` file. that way we avoid having to set them every single file